fluent-bit/files/bit.conf.j2

```diff
@@ -64,6 +64,13 @@
 
     {% macro plugin_config(plugins, type) -%}
     {%- for plugin in plugins %}
+      {%-
+        if type == 'OUTPUT'
+        and (plugin.get('Name', '').lower() == 'stackdriver' or plugin.get('name', '').lower() == 'stackdriver')
+        and not pillar.get('logging', {}).get('enabled', true)
```

Comment:
pillar items are not optional, change to `pillar['logging']['enabled']`.  otherwise we end up with a thousand default values scattered all over the codebase and it becomes hard to figure out why a value got applied somewhere.

Reply:
see above discussion with Keith. I changed it so that it's not optional but I eventually want to convert the `fluent_bit.sls` to a jinja config which will remove this hack.